src/utils/umm/gridKeyMap.json

```diff
@@ -0,0 +1,25 @@
+{
+  "sharedKeys": [
+    "conceptId",
+    "name"
+  ],
+  "ummKeyMappings": {
+    "additionalAttribute": "umm.AdditionalAttribute",
+    "conceptId": "meta.concept-id",
+    "contactMechanisms": "umm.Organization.ContactMechanisms",
+    "description": "umm.Description",
+    "dimensionSize": "umm.GridDefinition.DimensionSize",
+    "gridDefinition": "umm.GridDefinition",
+    "longName": "umm.LongName",
+    "metadataDate": "umm.MetadataDate",
+    "metadataSpecification": "umm.MetadataSpecification",
+    "name": "umm.Name",
+    "nativeId": "meta.nativeid",
```

Comment:
`nativeId` is not defined in `grid.graphql`

Reply:
Removed